components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/DBHandler.java

```diff
@@ -100,7 +100,7 @@ public boolean delete(PreparedStatement stmt) {
             stmt.close();
             return true;
         } catch (SQLException e) {
-            throw new RDBMSTableException(" Error while processing the dDELETE operation in " + stmt.toString(), e);
+            throw new RDBMSTableException(" Error while processing the dDELETE operation.", e);
```

Comment:
dDELETE?

Reply:
Fixed in 32f9bce1376e3d0a5f03e1723915ce3dd6c53df0